docs/docs/guides/web3_config/index.md

```diff
@@ -201,6 +202,575 @@ console.log(web3.getContextObject().config)
 */
 ```
 
+### handleRevert
+When `handleRevert` is set to True, the following methods will retrieve specific error types and error messages:
+```ts
+- web3.eth.sendTransaction();
+- web3.eth.call();
+- myContract.methods.myMethod().call();
+- myContract.methods.myMethod().send();
+```
+
+The error types will be one of the following:
+```ts
+- InvalidResponseError
+- ContractExecutionError
+- TransactionRevertWithCustomError
+- TransactionRevertedWithoutReasonError
+- TransactionRevertInstructionError
+- TransactionPollingTimeoutError
+```
+
+For example, the error message could be `TransactionRevertInstructionError('Returned error: invalid argument 0: json: cannot unmarshal invalid hex string into Go struct field TransactionArgs.data of type hexutil.Bytes')`. The default value is `false`, and handleRevert is only supported for `sendTransaction` and not for `sendSignedTransaction` for now.
+
+```ts
+import { Web3 } from 'web3';
+
+const web3 = new Web3('http://127.0.0.1:7545');
+
+web3.handleRevert = true;
+
+console.log(web3.getContextObject().config);
+```
+:::info
+The `handleRevert` can be configured both globally and at the package level:
+```ts
+import { Web3 } from 'web3';
+
+const web3 = new Web3('http://127.0.0.1:7545');
+
+web3.eth.handleRevert = true;
+
+console.log(web3.eth.getContextObject().config);
+```
+:::
+
+### defaultAccount
+This `defaultAccount` is used as the default `from` property, if no `from` property is specified in for the following methods:
+```ts
+- web3.eth.sendTransaction();
+- web3.eth.call();
+- myContract.methods.myMethod().call();
+- myContract.methods.myMethod().send();
+```
+
+The default value for `defaultAccount` is `undefined`.
```

Comment:
just like these lines. @luu-alex

Reply:
Yup! exactly how you've done it for these methods